gdxpds/gdx.py

```diff
@@ -440,11 +264,6 @@ def write(self,filename):
         if not ret:
             raise GdxError(self.H,"Could not open {} for writing. Consider cloning this file (.clone()) before trying to write".format(repr(filename)))
         self._filename = filename
-
-        # set special values
-        ret = gdxcc.gdxSetSpecialValues(self.H,self.special_values)
-        if ret == 0:
-            raise GdxError(self.H,"Unable to set special values")
```

Comment:
This code does not seem to have an equivalent in the refactor, which mostly seems okay, but I am wondering about your particular rationale @jebob.

Reply:
If we don't set the special values, we use the default values (`1e300`, `2e300` etc.).

It might be possible to skip conversion during writing if we setSpecialValues to `np.nan` etc. but I can't get it to read correctly.